lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -245,12 +245,15 @@ def _section_certificates(course):
             for cert_status in example_cert_status
         )
     )
+    instructor_generation_enabled = settings.FEATURES.get('CERTIFICATES_INSTRUCTOR_GENERATION', False)
```

Comment:
Nitpick: I think we try to use the naming convention ENABLE_xxxx for boolean features, but in this case, it starts to get a little long